### PR TITLE
add DUD support in bootloader_ofw.pm (for PowerPC)

### PR DESCRIPTION
**Pull request by michelmno**
* by default add insecure=1 if DUD is specified.

* also add kernel=1 and insecure=1 if NETBOOT but no SUSEMIRROR
  for investigation bug
  https://bugzilla.suse.com/show_bug.cgi?id=995367
  (note that this code is not executed on openQA main instance
  that always has the SUSEMIRROR).

Signed-off-by: Michel Normand <normand@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by michelmno** [comment]
I do not understand the travis-ci reported error versus my changes in bootloader_ofw.pm :(

**Comment by Soulofdestiny** [comment]
I guess you forgot to run "./tools/tidy" before commiting ;)

**Comment by michelmno** [comment]
still travis-ci error not related to the changes of this PR (changes only in bootloader_ofs.pm related to PowerPC).

**Comment by coolo** [comment]
don't get fooled by the colours - the errors is here:

```
Missing '# Summary: <multi line summary of test>' in tests/installation/bootloader_ofw.pm
Missing '# Maintainer: <email address>' in tests/installation/bootloader_ofw.pm

**Comment by michelmno** [comment]
@Soulofdestiny  and @coolo thanks for the info, I completed my changes.

**Comment by okurz** [comment]
Please write your git commit message subject lines starting with capital letter. We try to stick to  http://chris.beams.io/posts/git-commit/ here which also suggests this. And I am not sure on the use of `Signed-off-by` when you are also the author and commiter but I leave that to you :-)

File: tests/installation/bootloader_ofw.pm [hunk 1]
```diff
@@ -8,9 +8,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: Add first PowerPC test
-#    Signed-off-by: Dinar Valeev <dvaleev@suse.com>
-# G-Maintainer: Dinar Valeev <dvaleev@suse.com>
+# Summary: Add first PowerPC test
+# Maintainer: Dinar Valeev <k0da@opensuse.org>
```
Comment (Soulofdestiny) [inline tests/installation/bootloader_ofw.pm]: I don't think that Dinar still applies as maintainer, so maybe claim ownership for yourself? ;)

Comment (michelmno) [inline tests/installation/bootloader_ofw.pm]: @k0da  comment ?

Comment (k0da) [inline tests/installation/bootloader_ofw.pm]: I don't mind either keep or pass maintainership. I still have a time to help with failures, but it obviously is not my primary focus nowadays

Comment (Soulofdestiny) [inline tests/installation/bootloader_ofw.pm]: then it's fine with me

**Pull request merged**